2023-04-26 13:50:06 [1m[31mERROR[0m HTTPSConnectionPool(host='nohminatim.openstreetmap.org', port=443): Max retries exceeded with url: /search?street=pikk&city=tallin&county=+&state=+&country=+&postalcode=+&format=json (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000026AA58CABE0>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))